src/webextension/locales/fr_FR/widgets.ftl

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+copy-to-clipboard-button = Copie
+copy-to-clipboard-copied = ✔ Copié
+
+filter-input-clear
```

Comment:
Add = to the end of this line too. (I just noticed that I forgot to do this for this line in the `en-US` translation too, so I'll fix that version.)

Reply:
Fixed